include/rocksdb/utilities/backupable_db.h

```diff
@@ -274,16 +274,23 @@ class BackupEngineReadOnly {
     return RestoreDBFromLatestBackup(options, db_dir, wal_dir);
   }
 
+  // If verify_with_checksum is false, this function
```

Comment:
Maybe it is better to put the description of "True" first.

Reply:
Let me move the default case up :)